From 97d4b670a1261b4a51023e6bb16e622dd47f4b40 Mon Sep 17 00:00:00 2001
From: Russell Toris <rctoris@wpi.edu>
Date: Tue, 21 May 2013 13:44:50 -0400
Subject: [PATCH] postinst added

---
 debian/postinst |    3 +++
 1 file changed, 3 insertions(+)
 create mode 100755 debian/postinst

diff --git a/debian/postinst b/debian/postinst
new file mode 100755
index 0000000..ba13d63
--- /dev/null
+++ b/debian/postinst
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+sudo setcap cap_net_raw+ep /opt/ros/groovy/lib/youbot_oodl/youbot_oodl
\ No newline at end of file
-- 
1.7.9.5

