From bf27f377a24782e9126937211f2fee5f5d8d3401 Mon Sep 17 00:00:00 2001
From: Russell Toris <rctoris@wpi.edu>
Date: Wed, 22 May 2013 14:10:48 -0400
Subject: [PATCH 2/2] postinst script added

---
 debian/postinst |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/debian/postinst b/debian/postinst
index ba13d63..05fda81 100755
--- a/debian/postinst
+++ b/debian/postinst
@@ -1,3 +1,4 @@
 #!/bin/sh
 
-sudo setcap cap_net_raw+ep /opt/ros/groovy/lib/youbot_oodl/youbot_oodl
\ No newline at end of file
+echo "Running setcap for youbot_oodl..."
+setcap cap_net_raw+ep /opt/ros/groovy/lib/youbot_oodl/youbot_oodl
-- 
1.7.9.5

